### PR TITLE
Fixed RD-8890: com.amazonaws.AbortedException

### PR DESCRIPTION
**Pull request by bgaidioz**

### CODE REVIEW

File: raw-sources-s3/src/main/scala/raw/sources/filesystem/s3/RawS3InputStream.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+package raw.sources.filesystem.s3;
+
+import com.amazonaws.services.s3.model.S3ObjectInputStream;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InterruptedIOException;
+
+public class RawS3InputStream extends InputStream {
```
Comment (miguelbranco80) [inline raw-sources-s3/src/main/scala/raw/sources/filesystem/s3/RawS3InputStream.java:21]: Sorry, this is a Java file in a scala source folder. What's the reason?

Comment (bgaidioz) [reply raw-sources-s3/src/main/scala/raw/sources/filesystem/s3/RawS3InputStream.java]: I had two implementations while experimenting with exception handling, and didn't pay attention I kept the Java one.

File: raw-sources-s3/src/main/scala/raw/sources/filesystem/s3/RawS3InputStream.scala [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+package raw.sources.filesystem.s3
+
+import com.amazonaws.AbortedException
+import com.amazonaws.services.s3.model.S3ObjectInputStream
+
+import java.io.IOException
+import java.io.InputStream
+import java.io.InterruptedIOException
+
+class RawS3InputStream(private var s3ObjectInputStream: S3ObjectInputStream) extends InputStream {
```
Comment (miguelbranco80) [inline raw-sources-s3/src/main/scala/raw/sources/filesystem/s3/RawS3InputStream.scala:22]: Should be a private val I think.

**Review by miguelbranco80 — APPROVED** [review]

File: raw-sources-s3/src/main/scala/raw/sources/filesystem/s3/RawS3InputStream.scala (hunk 2 above)
Comment (bgaidioz) [reply raw-sources-s3/src/main/scala/raw/sources/filesystem/s3/RawS3InputStream.scala]: I relied on the Java to Scala translation and didn't see that. I think it's OK without `private val`.

**Pull request merged**